src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -4625,6 +4897,14 @@ components:
       required: true
       schema:
         type: string
+    XactId:
+      name: xactId
+      in: path
+      description: A UUID to used to identify a transaction in a dataset
+      required: true
+      schema:
+        type: string
+        format: uuid
```

Comment:
I think you mentioned you were renaming the short hand "xact" to transaction. If so, I'm seeing 11 instances of xactId across these changes.

Reply:
shoot, you're right. I'll do a grep pass

Reply:
Also, @nmalfroy  - friendly reminder for this change.

Reply:
Good call.  Updated!